src/frontend/src/sass/_buttons.scss

```diff
@@ -132,6 +132,7 @@ a:hover {
 .bcgovbtn__tertiary {
   --text-color: #{$primary-colour};
 
+  padding-top: 1.2em;
```

Comment:
adding padding-top to *all* secondary buttons isn't a way to go. Can you please come up with a neater solution.

Note: this impacts position of `List of PIAs` navigation buttons and other secondary btn usages.